app/locale/en.coffee

```diff
@@ -1294,13 +1293,17 @@
     assign_to_selected_students: "Assign to Selected Students"
     assigned: "Assigned"
     enroll_selected_students: "Enroll Selected Students"
+    cant_assign_to_unenrolled: "Course cannot be assigned to students who are not enrolled."
+    no_students_selected: "No students were selected."
 
     guides_coming_soon: "Guides coming soon!" # Courses
 
     show_students_from: "Show students from" # Enroll students modal
     enroll_the_following_students: "Enroll the following students"
     all_students: "All Students"
-
+    enroll_students: "Enroll Students"
+    
```

Comment:
I'm getting rid of extra newlines within sections so that the line numbers in the other generated locale files will line up with the ones in en.coffee (when they've been propagated, anyway).